sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/client.py

```diff
@@ -168,9 +168,7 @@ def __init__(self, hostname, **kwargs):
         self._mgmt_links = {}
         self._mgmt_link_lock = threading.Lock()
         self._retry_policy = kwargs.pop("retry_policy", RetryPolicy())
-        self._keep_alive_interval = kwargs.get("keep_alive_interval", 0)
-        self._keep_alive_interval = int(self._keep_alive_interval) if self._keep_alive_interval is not None else 0
```

Comment:
changed this

Reply:
`self._keep_alive_interval = int(kwargs.get("keep_alive_interval") or 0)` should work if we want to keep it one line

Reply:
no will revert

Reply:
> `self._keep_alive_interval = int(kwargs.get("keep_alive_interval") or 0)` should work if we want to keep it one line

That would raise too if kwargs["keep_alive_interval"] = None.

Reply:
That would eval to int(None or 0) --> int(0)